### PR TITLE
Dynamically get app settings as per openmrs application distribution.

### PR DESCRIPTION
**Pull request by paulupendo**
## JIRA TICKET NAME:
[CB-48 click here to view on JIRA](https://issues.openmrs.org/browse/OWA-48)
### SUMMARY:
- Consumes 'ws/rest/owa/settings'  for manageSettings component. 
- Adds xml to son converter

### CODE REVIEW

File: app/js/components/homePage/manageSettings.jsx [hunk 1]
```diff
@@ -18,30 +19,60 @@ export default class ManageSettings extends React.Component {
...
+    this.setDefault = this.setDefault.bind(this);
   }
 
   componentWillMount() {
-    this.setState({
-      AppFolderPathValue: 'appdata/owa',
-      AppStoreURLValue: 'http://modules.openmrs.org'
+    const settingsPaths = this.settingsData();
+    this.setState({ 
+      AppFolderPathValue: settingsPaths != undefined ? settingsPaths[0].propertyValue : 'appdata\\owa',
```
Comment (dkayiwa) [inline app/js/components/homePage/manageSettings.jsx]: why back ward slashes?

File: app/js/components/homePage/manageSettings.jsx [hunk 2]
```diff
@@ -18,30 +19,60 @@ export default class ManageSettings extends React.Component {
...
-  onCancel() {
+
+  setDefault() {
     event.preventDefault();
+    const settingsPaths = this.settingsData();
     this.setState({ 
-      AppBaseURLValue: '',
-      AppFolderPathValue: 'appdata/owa',
-      AppStoreURLValue: 'http://modules.openmrs.org'
+      AppFolderPathValue: settingsPaths != undefined ? settingsPaths[0].propertyValue : 'appdata\\owa',
```
Comment (dkayiwa) [inline app/js/components/homePage/manageSettings.jsx]: why backward slashes?

**Comment by paulupendo** [comment]
@dkayiwa The folder to the AppFolderPath is named with the escape character. I have parsed the raw string to avoid the double backslashes used before but keep the correct folder name.

File: app/js/components/homePage/manageSettings.jsx [hunk 3]
```diff
@@ -18,30 +19,61 @@ export default class ManageSettings extends React.Component {
...
+
+    parser(data, function(err, result){
+      jsonData = result;
+    });
+
+    try {
+      return jsonData.list["org.openmrs.GlobalProperty"];
+    } catch(e) {
+      if(e instanceof TypeError) {
+        console.log("An Error Occured");
```
Comment (jomadoye) [inline app/js/components/homePage/manageSettings.jsx]: @paulupendo `console.log()` should be avoided.

**Comment by jomadoye** [comment]
Awesome job @paulupendo

**Comment by wanjikum** [comment]
Goodwork @paulupendo !

**Pull request merged**